src/azure-cli/requirements.py3.windows.txt

```diff
@@ -116,6 +116,7 @@ pbr==5.3.1
 portalocker==1.7.1
 psutil==5.8.0
 pycparser==2.19
+PyGithub==1.38
```

Comment:
Also add it to `setup.py`

Reply:
Added. Why are only some of the requirements.txt added to setup.py dependencies?

Reply:
`pip install` reads `setup.py`. Windows MSI artifact build reads `requirements.txt`. It also makes me confused why two dependency definitions.
@fengzhou-msft Could you answer the question by Calvin?